HeterogeneousCore/CUDAServices/plugins/NVProfilerService.cc

```diff
@@ -543,6 +541,11 @@ void NVProfilerService::postBeginJob() {
   }
 }
 
+void NVProfilerService::lookupInitializationComplete(edm::PathsAndConsumesOfModulesBase const& pathsAndConsumes,
+                                                     edm::ProcessContext const& context) {
+  callgraph_.lookupInitializationComplete(pathsAndConsumes, context);
```

Comment:
@wddgit could you add a marker here ?
```c++
  nvtxDomainMark(global_domain_, "lookupInitializationComplete");
```

Or, if you think that displaying some information from the `lookupInitializationComplete` signal could be interesting, something like
```c++
  std::stringstream out;
  out << "lookupInitializationComplete: " << ...;
  nvtxDomainMark(global_domain_, out.str().c_str());
```

Reply:
Done. I added the call to nvtxDomainMark.